### DISCUSSION TITLE
bigfix/11.0/platform/Platform/Web_Reports/c_enabling_sec_policy

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#90] [Q&A]
# bigfix/11.0/platform/Platform/Web_Reports/c_enabling_sec_policy

In the Web Report component, you can use a new option named "EnableCSPHeader" to improve the custom report security.

https://help.hcl-software.com/bigfix/11.0/platform/Platform/Web_Reports/c_enabling_sec_policy.html

**Comment by Jwalker107** [comment]
It is not specified in this article but the value type of EnableCSPHeader should be type REG_SZ for a string value, not a REG_DWORD or other type.
